Salief/sunday read

salieflewis opened the pull request.

0xTranqui commented:
thanks for the cleanup. 2 small nits before merging if ur down to fix:

1. i moved the compiler to 0.8.19 but am seeing the vs code settings are still showing up as 0.8.17. could u update that?
2. (see below for image) - i think best practice is to avoid duplicate identical names like this, even if it works. maybe add the _ before the river argument input? or come up with a new name to use for it? 

<img width="507" alt="Screenshot 2023-08-21 at 12 21 12 PM" src="https://github.com/public-assembly/assembly-press/assets/93691906/49f91cff-e536-4a04-8a03-2827289200ff">

salieflewis commented:
I changed the compiler back to 'remote' which I think should solve the issues you were having, but also is now giving me errors on my machine. Not really sure how to properly go about this, can we setup something in our `foundry.toml`.

And I changed `router` to be `routerAddr` in the context of the Press. I think I changed this everywhere it needs to be, and also updated it in the context of the Press interface which previously I had neglected to do.

cc: @0xTranqui
